controller/api/destination/endpoint_translator.go

```diff
@@ -373,17 +378,26 @@ func (et *endpointTranslator) sendClientAdd(set watcher.AddressSet) {
 		if address.Pod != nil {
 			opaquePorts = watcher.GetAnnotatedOpaquePorts(address.Pod, et.defaultOpaquePorts)
 			wa, err = createWeightedAddr(address, opaquePorts, et.enableH2Upgrade, et.identityTrustDomain, et.controllerNS, et.log)
+			if err != nil {
```

Comment:
is pulling this error handling into both branches of the if a functional change?  what's the effect of this change?

Reply:
there's no functional change and it's not required -- just jarring to continue processing things even when we've received an error